### ISSUE TITLE
Post-install fails on windows because unzip missing

### ISSUE THREAD

**Issue by spolu** [breach/node-thrust#7]
See https://github.com/breach/thrust/issues/227

```
events.js:72
throw er; // Unhandled 'error' event
^
Error: spawn ENOENT
at errnoException (child_process.js:1001:11)
at Process.ChildProcess._handle.onexit (child_process.js:792:34)
npm ERR! node-thrust@0.7.5 postinstall: node scripts/bootstrap.js
npm ERR! Exit status 8
```

**Comment by morganrallen** [comment]
https://www.npmjs.org/package/unzip

**Comment by spolu** [comment]
Should be fixed and has been published with `0.7.5-rc2`
